### PR TITLE
Standardising mavlink message strings

### PR DESCRIPTION
**Pull request by potaito**
I tried to standardise the mavlink message strings a tiny bit:
* No ALL CAPS words (even when the situation is really bad. But let's face it: A lot of messages can indicate something really bad)
* Begin message with a capital letter
* No punctuation character at end of message
* Minor English adjustments

I understand that this is a matter of taste to some extent, but let's see :)

### CODE REVIEW

File: src/modules/commander/Commander.cpp [hunk 1]
```diff
@@ -1038,7 +1038,7 @@ Commander::handle_command(vehicle_status_s *status_local, const vehicle_command_
 			// if no high latency telemetry exists send a failed acknowledge
 			if (_high_latency_datalink_heartbeat > commander_boot_timestamp) {
 				cmd_result = vehicle_command_s::VEHICLE_CMD_RESULT_FAILED;
-				mavlink_log_critical(&mavlink_log_pub, "Control high latency failed, no hl telemetry available");
```
Comment (potaito) [inline src/modules/commander/Commander.cpp]: This guy exceeded 50 characters before the change.

File: src/modules/commander/Commander.cpp [hunk 2]
```diff
@@ -2118,7 +2118,7 @@ Commander::run()
 
 		} else {
 			if (!status_flags.rc_input_blocked && !status.rc_signal_lost) {
-				mavlink_log_critical(&mavlink_log_pub, "MANUAL CONTROL LOST (at t=%" PRIu64 "ms)", hrt_absolute_time() / 1000);
+				mavlink_log_critical(&mavlink_log_pub, "Manual control lost (at t=%" PRIu64 "ms)", hrt_absolute_time() / 1000);
```
Comment (julianoes) [inline src/modules/commander/Commander.cpp]: In my opinion we should remove the timestamp here.

Comment (potaito) [reply src/modules/commander/Commander.cpp]: Fine by me. I always wondered why this message should be exceptional :sweat_smile:

File: src/modules/commander/Commander.cpp [hunk 3]
```diff
@@ -3997,11 +3997,11 @@ void Commander::data_link_check(bool &status_changed)
 			}
 
 			if (_datalink_last_status_avoidance_system == telemetry_status_s::MAV_STATE_CRITICAL) {
-				mavlink_log_info(&mavlink_log_pub, "Avoidance system timeout");
+				mavlink_log_info(&mavlink_log_pub, "Avoidance system timed out");
 			}
 
 			if (_datalink_last_status_avoidance_system == telemetry_status_s::MAV_STATE_FLIGHT_TERMINATION) {
-				mavlink_log_critical(&mavlink_log_pub, "Avoidance system abort");
+				mavlink_log_critical(&mavlink_log_pub, "Avoidance system aborted");
```
Comment (potaito) [inline src/modules/commander/Commander.cpp]: @mrivi I assume you know this: What's the meaning of `aborted` here? Is autonomous flight terminated and the flight controller stops listening to obstacle avoidance? Or is something wrong with the obstacle avoidance? I assume it's the former and not the latter. :)

Comment (mrivi) [reply src/modules/commander/Commander.cpp]: It means that there is something wrong with the avoidance and the vehicle doesn't accept anymore any setpoint from the OA system

Comment (potaito) [reply src/modules/commander/Commander.cpp]: Should we then say "Avoidance system rejected" ?

Comment (mrivi) [reply src/modules/commander/Commander.cpp]: Fine for me

**Comment by hamishwillee** [comment]
Late to the party. This is great!

**Comment by potaito** [comment]
Thanks @hamishwillee :grin:

**Pull request merged**